behat.yml.dist

```diff
@@ -55,7 +55,33 @@ default:
       javascript_session: 'selenium2'
       selenium2:
         wd_host: '${selenium.host}:${selenium.port}/wd/hub'
-        capabilities: { "browser": "chrome", "version": "*" }
+        capabilities:
```

Comment:
Let's start with the minimum set used by QA at https://webgate.ec.europa.eu/fpfis/qa/how-to?search=selenium:
```
capabilities:
  nativeEvents: true
  marionette: true
  browserName: chrome
```